bin/micro

```diff
@@ -28,7 +29,8 @@ const help = () => {
   console.log(`Usage: micro [opts] <file>
   -H, --host  Host to listen on   [0.0.0.0]
   -p, --port  Port to listen on      [3000]
-  -h, --help  Show this help message`);
+  -h, --help  Show this help message
+  -L, --log dev|prod|off`);
```

Comment:
Turned this around in my merge commit. First the logging then help, also added `[off]` as default indicator